app/api/routes.py

```diff
@@ -35,7 +37,9 @@ def get_resources():
 def get_languages():
     try:
         page = request.args.get('page', 1, type=int)
-        language_paginator = Language.query.paginate(page, Config.LANGUAGES_PER_PAGE, False)
+        page_size = request.args.get('page_size', Config.LANGUAGES_PER_PAGE, type=int)
```

Comment:
I like the use of type setting here. If someone passes in `?page=bcd2` we'll crash. 

What do you think should happen?

Reply:
Either we should use the default if something unexpected is encountered, or we should return an error code that says "invalid_value" with a message that explains that `bcd2 is invalid` or something. But I like the former, because if the client side programmer does something wrong, the user doesn't suffer because of it.

What do you think?

Reply:
Also, I just passed in `bcd2` and it didn't crash, it returned the default number of resources. We can conduct more extensive tests, but I suspect that the behavior we're getting is actually what I described as my preference above.